steps/browser_steps.py

```diff
@@ -75,9 +77,17 @@ def step_impl(context, text):
     :type context: behave.runner.Context
     :type text: str
     """
-    context.browser.get_current_page().get_text().should.match(r'.*{}.*'.format(text))
+    context.browser.page.get_text().should.match(r'.*{}.*'.format(text))
```

Comment:
`get_current_page()` and `page` currently provide access to the same thing, but `get_current_page()` may go away in the future